_episodes/05-accelerating-lammps.md

```diff
@@ -118,8 +118,13 @@ pair style (say, `reax`) for which this is not implemented, its obvious that you
 going to have a performance gain for the `Pair` part of the calculation. If the
 majority of the computation time is coming from the `Pair` part then you are in trouble.
 If you would like to know how much speedup you can expect to achieve using USER-INTEL, you can
-take a look in the
-[corresponding LAMMPS documentation](https://lammps.sandia.gov/doc/Speed_intel.html)
+take a look in the corresponding
+[LAMMPS documentation](https://lammps.sandia.gov/doc/Speed_intel.html).
+
+> ## Single, double, mixed precision calculations
+>
+> ***NEED TO EXPLAIN WHAT THIS MEANS IN CALLOUT*** Explain me!
```

Comment:
@prithwish-ichec ignore previous, please elaborate on `Single, double and mixed precision calculations`. How does it relate to USER-INTEL package, what do these mean?

Reply:
to be done

Reply:
Will leave to be staged in next merge.